app/bundles/LeadBundle/Model/LeadModel.php

```diff
@@ -1516,6 +1515,31 @@ public function importLead($fields, $data, $owner = null, $list = null, $tags =
             );
         }
 
+        $booleanTrue  = ['1', 'true', 'yes'];
```

Comment:
This could be simplified with `$value = (int) filter_var($fieldData[$leadField['alias'], FILTER_VALIDATE_BOOLEAN);`

http://php.net/manual/en/filter.filters.validate.php
```
Returns TRUE for "1", "true", "on" and "yes". Returns FALSE otherwise.

If FILTER_NULL_ON_FAILURE is set, FALSE is returned only for "0", "false", "off", "no", and "", and NULL is returned for all non-boolean values.
```